modules/understanding-upgrade-channels.adoc

```diff
@@ -24,6 +24,7 @@ binary file for a specific version of {product-title} always installs that versi
 * `candidate-{product-version}`
 * `fast-{product-version}`
 * `stable-{product-version}`
+* `eus-4.6` (only available when running 4.6)
```

Comment:
This should say, when running an EUS version, as we will have more EUS versions than just 4.6

Reply:
This is accurate as written, you will not see the eus-4.6 channel unless you're already running 4.6 versions and we don't have to worry about other eus channels until next November, we can deal with addition of subsequent EUS channels then.